website/common/script/libs/taskDefaults.js

```diff
@@ -66,7 +66,7 @@ export default function taskDefaults (task, user) {
   }
 
   if (task.type === 'daily') {
-    const now = moment().zone(user.preferences.timezoneOffset);
+    const now = moment().utcOffset(-user.preferences.timezoneOffset);
```

Comment:
Similar to sanitizeOptions() in cron,
it might be a good idea to provide a common lib function getUtcOffset(user).

Reply:
Yeah good point. Will implement tonight.

Reply:
Done.